worlds/AutoWorld.py

```diff
@@ -115,6 +119,10 @@ def call_single(multiworld: "MultiWorld", method_name: str, player: int, *args:
             logging.error(message)
         raise e
     else:
+        taken = time.perf_counter()-start
+        if taken > 0.1:
+            perf_logger.info(f"Took {taken} seconds in {method} for player {player}, "
```

Comment:
```suggestion
            perf_logger.info(f"Took {taken} seconds in {multiworld.worlds[player].__class__.__name__}'s {method.__name__} for player {player}.")
``` maybe?

Reply:
I'd be fine with cutting away the ram address, but you're cutting away too much. It also would lose consistency with the error reporting.

Reply:
have you tried qualname? Because all it is is "World.set_rules", no matter which world

Reply:
tested on 3.11.6 and i'm seeing the same output. one interesting thing to note is that it doesn't show the class for methods where it's assigned rather than overridden, since those functions don't belong to the class. ![Screenshot_36](https://github.com/ArchipelagoMW/Archipelago/assets/13184667/4c8cb66f-896c-4880-9fe6-988d3083e473)